docs/sources/tutorials/alerting-get-started-pt3/index.md

```diff
@@ -0,0 +1,446 @@
+
+     **Timing options** control how often notifications are sent and can help balance timely alerting with minimizing noise.
+
+1. Save and repeat:
+
+   - Repeat for `region = us-east` with a different webhook or a different contact point.
+
+   {{< figure src="/media/docs/alerting/notificaiton-policies-region.png" max-width="750px" alt="Two nested notification policies to route and group alert notifications" >}}
+
+   These nested policies should route alert instances where the region label is either us-west or us-east.
+
+   **Note**: In Grafana, each label within a notification policy must have a unique key. If you attempt to add the same label key (e.g., region) with different values (us-west and us-east), only the last entry will be saved, and the previous one will be discarded. This is because labels are stored as associative arrays (maps), where each key must be unique.
+   For identical label keys use regex matchers (e.g., _region=~"us-west|us-east"_).
+
+<!-- INTERACTIVE ignore END -->
+
+{{< docs/ignore >}}
+
+1. Visit [http://localhost:3000](http://localhost:3000), where Grafana should be running
+1. Navigate to **Alerts & IRM > Alerting > Notification policies**.
+1. In the Default policy, click **+ New child policy**.
+
+   - In the Default policy, click **+ New child policy**.
+   - **Label**: `region`
+   - **Operator**: `=`
+   - **Value**: `us-west`
+
+   This label matches alert rules where the region label is us-west
+
+1. Choose a **Contact point**:
+
+   - Select **Webhook**.
+
+   If you don’t have any contact points, add a Contact point.
+
+1. Enable Continue matching:
+
+   - Turn on **Continue matching subsequent sibling nodes** so the evaluation continues even after one or more labels (i.e. region label) match.
+
+1. Override grouping settings:
+
+   - Toggle **Override grouping**.
+   - **Group by**: `region`.
+
+     **Group by** consolidates alerts that share the same grouping label into a single notification. For example, all alerts with `region=us-west` will be combined into one notification, making it easier to manage and reducing alert fatigue.
+
+1. Set custom timing:
+
+   - Toggle **Override general timings**.
+   - **Group interval**: `2m`. This ensures follow-up notifications for the same alert group will be sent at intervals of 2 minutes.
+
+     **Timing options** control how often notifications are sent and can help balance timely alerting with minimizing noise.
+
+1. Save and repeat:
+
+   - Repeat for `region = us-east` with a different webhook or a different contact point.
+
+     **Note**: In Grafana, each label within a notification policy must have a unique key. If you attempt to add the same label key (e.g., region) with different values (us-west and us-east), only the last entry will be saved, and the previous one will be discarded. This is because labels are stored as associative arrays (maps), where each key must be unique.
```

Comment:
```suggestion
     **Note**: In Grafana, each label within a notification policy must have a unique key. If you attempt to add the same label key (e.g., region) with different values (us-west and us-east), only the last entry is saved, and the previous one is discarded. This is because labels are stored as associative arrays (maps), where each key must be unique.
```